root.tf

```diff
@@ -67,6 +67,28 @@ module "ingest_parsed_court_documents_event_handler_lambda" {
   }
 }
 
+module "ingest_mapper_lambda" {
+  source          = "git::https://github.com/nationalarchives/da-terraform-modules//github_repository_secrets"
+  repository_name = "nationalarchives/dr2-ingest-mapper"
+  secrets = {
+    MANAGEMENT_ACCOUNT = data.aws_caller_identity.current.account_id
+    SLACK_WEBHOOK      = data.aws_ssm_parameter.github_slack_webhook.value
+    WORKFLOW_TOKEN     = data.aws_ssm_parameter.github_workflow_token.value
+  }
+}
+
+module "ingest_mapper_environments" {
+  for_each              = module.configuration.account_numbers
+  source                = "git::https://github.com/nationalarchives/da-terraform-modules//github_environment_secrets"
+  environment           = each.key
+  repository_name       = "nationalarchives/dr2-ingest-mapper"
+  team_slug             = "digital-records-repository"
+  integration_team_slug = []
+  secrets = {
+    ACCOUNT_NUMBER = each.value
+  }
+}
+
 module "ingest_parsed_court_documents_event_handler_environments" {
```

Comment:
Might be best to move this directly below `"ingest_parsed_court_documents_event_handler_lambda"`; your new changes split them in half

Reply:
Makes sense, I've moved it.